### PR TITLE
Extract zonation from grid

### PR DESCRIPTION
**Pull request by tnatt**

### CODE REVIEW

File: src/fmu/tools/qcforward/extract_grid_zone_tops.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+):
+    """
+    Function for extracting top and base from gridzones, both in TVD and MD.
+
+    Users can either input a pre-generated gridzonelog or a grid and a zone parameter
+    for computing the gridzonelog.
+
+    The function works both inside RMS and outside with file input. If input from files,
+    and a MD log is not present in the well a quasi md log will be computed and used.
+    """
```
Comment (HansKallekleiv) [inline src/fmu/tools/qcforward/extract_grid_zone_tops.py:23]: Describe what the function returns

File: tests/test_extract_zone_tops.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import pytest
+
+from fmu.tools.qcforward.extract_grid_zone_tops import extract_grid_zone_tops
+
+
+if "__file__" in globals():
+    # Easen up copying test code into interactive sessions
+    testdir = os.path.dirname(os.path.abspath(__file__))
+else:
+    testdir = os.path.abspath(".")
```
Comment (jcrivenaes) [inline tests/test_extract_zone_tops.py:12]: What is this doing?

File: tests/test_extract_zone_tops.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+
+GRID = os.path.join(testdir, "data/reek_sim_grid.roff")
+GRIDPROP = os.path.join(testdir, "data/reek_sim_zone.roff")
```
Comment (jcrivenaes) [inline tests/test_extract_zone_tops.py:16]: Rather use xtgeo testrepo directly, as ../xtgeo-testdata/... so we avoid filling this repo with large chunks of tesdata

File: tests/test_extract_zone_tops.py (hunk 2 above)
Comment (HansKallekleiv) [reply tests/test_extract_zone_tops.py:12]: @berland ?

**Review by HansKallekleiv — APPROVED** [review]
lgtm!

**Review by jcrivenaes — APPROVED** [review]

File: src/fmu/tools/qcforward/extract_grid_zone_tops.py [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+def shift_zone_values(zvals):
+    for idx, _zval in enumerate(zvals):
+        if idx == len(zvals) - 1:
+            continue
+        if zvals[idx] == zvals[idx + 1]:
+            zvals[idx + 1] = zvals[idx + 1] - 1
+    return zvals
+
+
+def make_alias_dict(alias_file, rms_name="RMS_WELL_NAME", ecl_name="ECLIPSE_WELL_NAME"):
```
Comment (berland) [inline src/fmu/tools/qcforward/extract_grid_zone_tops.py:117]: I have found exactly these codelines in create_ert_rftobs.py that I am refactoring. That means that this alias/file/dict functionality should be taken out of these individual codes and put somewhere smart.

File: .github/workflows/fmu-tools.yml [hunk 5]
```diff
@@ -50,6 +50,7 @@ jobs:
         pip install .
 
     - name: Run tests
+      if: matrix.python-version != '2.7'
```
Comment (berland) [inline .github/workflows/fmu-tools.yml:53]: delete

**Pull request merged**